Jropp/jason-ropp-js-iobp
# Popup messages

## What It Does

When a save is completed, the database API dispatched event that notifies `user-list.html` to update users, includes a message of the action that was taken and completed. 

This message is sent to the popupMessage function, which sets the text content of the message box. It also sets the class of the message box to be visible for three seconds.

```
popupMessage(action) {
        let messageBox = this.$.popupBox;
        messageBox.textContent = action + ' ' + 'Successful';
        messageBox.className = 'message-box show-message';
        window.setTimeout(() => {
          messageBox.className = 'message-box';
        }, 3000);
      }
```

## When Completed

- [ ] Styles should mimic JHA-toast messages
- [ ] Popup should use CSS animations
- [ ] The message should popup and then disappear in the bottom left corner after three seconds.
- [ ] Proper message should be displayed that match click events.


## How To Test

Do any save or delete action. The message should popup and then disappear after three seconds.

## Notes/Caveats

## Dependencies

#68
